MapboxNavigation/NavigationCustomizable.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+import MapboxCoreNavigation
+
+/**
+ The `NavigationCustomizable` protocol represents a UI-based mechanism that allows for customization of its visual style, as well as the navigation service that powers it.
+ */
+protocol NavigationCustomizable {
+    /**
+     The styles that the User Interface can select from for display.
```

Comment:
Nit: UI is capitalized, but user interface is not, since it’s a common noun.

```suggestion
     The styles that the user interface can select from for display.
```